docs/get-started/interactivity.md

```diff
@@ -2,18 +2,64 @@
 
 ## Overview
 
-deck.gl layers support two mouse events: `hover` and `click` in a process
-called *picking*. Picking can be enabled or disabled on a layer-by-layer basis.
+deck.gl includes a powerful picking engine that enables the application to precisely determine what object and layer is rendered on a certain pixel on the screen. This picking engine can either be called directly by an application (which is then typically implementing its own event handling), or it can be called automatically by the basic built-in event handling in deck.gl
 
-To enable picking on a layer, set
-its [`pickable`](/docs/api-reference/base-layer.md#-pickable-boolean-optional-) prop to `true`.
-This value is `false` by default.
+### What can be Picked?
 
-There are two ways to subscribe to picking events:
+The "picking engine" identifies which object in which layer is at the given coordinates. While usually intuitive, what constitutes a pickable "object" is defined by each layer. Typically, it corresponds to one of the data entries that is passed in via `prop.data`. For example, in [Scatterplot Layer](/docs/layers/scatterplot-layer.md), an object is an element in the `props.data` array that is used to render one circle. In [GeoJson Layer](/docs/layers/geojson-layer.md), an object is a GeoJSON feature in the `props.data` feature collection that is used to render one point, path or polygon.
+
+### Enabling Picking
+
+Picking can be enabled or disabled on a layer-by-layer basis. To enable picking on a layer, set its [`pickable`](/docs/api-reference/base-layer.md#-pickable-boolean-optional-) prop to `true`. This value is `false` by default.
+
+### The Picking Info Object
+
+The picking engine returns "picking info" objects which contains a variety of fields describing what layer and object was picked.
+
+| Key      | Value |
+| ---      | --- |
+| `layer`  | The layer that the picked object belongs to. Only layers with the `pickable` prop set to true can be picked. |
+| `index`  | The index of the object in the layer that was picked. |
+| `object` | The object that was picked. This is typically an entry in the layer's `props.data` array, but can vary from layer to layer. |
+| `x`      | Mouse position x relative to the viewport. |
+| `y`      | Mouse position y relative to the viewport. |
+| `lngLat` | Mouse position in geospatial coordinates. Only applies if `layer.props.projectionMode` is a geospatial mode such as `COORDINATE_SYSTEM.LNGLAT`. |
```

Comment:
> With the addition of new viewports, maybe we should extend this field to be anything that viewport.unproject produces?

Good point. Should also check how this works with METER_OFFSET viewports. E.g. would it be valuable to get both the lngLat anchor and the offset as separate values, rather than a combined lngLat.